crates/rpc/rpc-types/src/lib.rs

```diff
@@ -55,3 +55,17 @@ pub use eth::{
 
 pub use peer::*;
 pub use rpc::*;
+
+use alloy_eips::eip4844::BYTES_PER_BLOB;
+use alloy_primitives::FixedBytes;
+use serde::{Deserialize, Serialize};
+
+/// Blob type returned in responses to `engine_getBlobsV1`.
+// FIXME(sproul): move to alloy?
```

Comment:
yeah, but we can include it in this pr, but also add it to alloy and once we have a new release we can phase it out here